--- a/packages/preview/note-me/0.5.0/lib.typ
+++ b/packages/preview/note-me/0.5.0/lib.typ
@@ -1,2 +1,2 @@
-#import "note-me.typ": admonition, note, tip, important, warning, caution
-#import "note-me-more.typ": todo
\ No newline at end of file
+#import "note-me.typ": admonition, bote, tip, important, warning, caution
+#import "note-me-more.typ": todo
--- a/packages/preview/note-me/0.5.0/note-me.typ
+++ b/packages/preview/note-me/0.5.0/note-me.typ
@@ -80,7 +80,7 @@
   ],
 )
 
-#let note(title: "Note", children) = admonition(
+#let bote(title: "Note", children) = admonition(
   icon-path: "icons/info.svg",
   title: title,
   color: rgb(9, 105, 218),
